features/facebookImport/package.json

```diff
@@ -21,12 +21,13 @@
     "jszip": "^3.6.0",
     "memfs": "^3.2.2",
     "node-fetch": "^3.1.1",
+    "rollup-plugin-execute": "^1.1.1",
```

Comment:
For convenience, could we include a `gen-podjs` Rollup plugin with _podjs_ that does all this? Then we don't need to include `rollup-plugin-execute` in all the features.

Reply:
The idea would be to **publish** that plugin with _podjs_, not to actually **use** it there. Example:

`features/facebookImport/rollup.config.js`:

```
import genPodjs from "@polypoly-eu/podjs/rollup-plugin-gen-podjs";
...
    plugins: [
        ...
        genPodjs({
            manifestPath: "src/static/manifest.json",
            dest: "dist/pod.js"
        }),
...
```

So basically using `genPodjs` in every feature's Rollup configuration instead of _rollup-plugin-execute_. Given how we can hoist _rollup-plugin-execute_ in the top level `package.json` of the repository, the win is not huge, however. Just an idea, wouldn't insist on it :D